src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.repackaged.com.google.common.io.CharStreams;
+import com.google.common.collect.ImmutableList;
+import com.google.gson.Gson;
+import com.googleinterns.zoomtube.data.Comment;
+import java.io.IOException;
+import java.util.Date;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Manages discussions for a lecture.
+ */
+@WebServlet("/discussion")
+public class DiscussionServlet extends HttpServlet {
+  private static final String PARAM_LECTURE = "lecture";
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long lectureId = Long.parseLong(request.getParameter(PARAM_LECTURE));
+    Key lecture = KeyFactory.createKey(/* kind= */ "Lecture", lectureId);
```

Comment:
I realize "Lecture" is used multiple times. Please create a private static final String LECTURE_KEY for it. Then you can omit the inline comment.

Reply:
Once Jay's lecture stuff is in master I'm going to use `Lecture.ENTITY_KIND` until the lecture stuff is added the values here really don't do anything

Reply:
Added TODO